packages/validator-bonds-cli-core/src/commands/manage/mintBond.ts

```diff
@@ -33,18 +33,26 @@ export function configureMintBond(program: Command) {
       'Rent payer for the mint token account creation (default: wallet keypair)',
       parseWalletOrPubkey,
     )
+    .option(
+      '--compute-unit-limit <number>',
+      'Compute unit limit for the transaction (default value based on the operation type)',
```

Comment:
```suggestion
      'Compute unit limit for the transaction',
```
(non-blocking)
This seems to have a specific default value of `MINT_BOND_LIMIT_UNITS` instead of being based on the operation type, so this might be a copy-paste error.